### PR TITLE
fix: prod deployment

### PR DESCRIPTION
**Pull request by Th0rgal**
This pull request fixes the deployment in production.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -18,54 +18,39 @@ services:
         - SEARXNG_API_URL=http://searxng:8080
     depends_on:
       - searxng
-      - mongodb
-      - cairobook-ingest
     ports:
       - 3001:3001
     volumes:
+      - ./src:/home/perplexica/src
+      - ./config.toml:/home/perplexica/config.toml
```
Comment (enitrat) [inline docker-compose.yaml]: normally the docker image covers that. I don't think it's needed

File: app.dockerfile [hunk 2]
```diff
@@ -6,10 +6,8 @@ ENV NEXT_PUBLIC_WS_URL=${NEXT_PUBLIC_WS_URL}
 ENV NEXT_PUBLIC_API_URL=${NEXT_PUBLIC_API_URL}
 
 WORKDIR /home/perplexica
-
-COPY ui /home/perplexica/
+COPY ui ./
```
Comment (enitrat) [inline app.dockerfile:9]: are you sure for this one ?

File: docker-compose.yaml [hunk 3]
```diff
@@ -18,54 +18,39 @@ services:
...
       - backend-dbstore:/home/perplexica/data
     extra_hosts:
       - 'host.docker.internal:host-gateway'
     networks:
       - perplexica-network
-    restart: unless-stopped
+    environment:
+      - NODE_ENV=production
+      - NEXT_PUBLIC_HOSTED_MODE=true
```
Comment (enitrat) [inline docker-compose.yaml]:
```suggestion
```

normally this depends on the config.toml so no need for that env var here

File: app.dockerfile (hunk 2 above)
Comment (Th0rgal) [reply app.dockerfile:9]: yes

**Pull request merged**